.github/workflows/ci-rust.yml

```diff
@@ -0,0 +1,98 @@
+name: 'CI - Rust'
+on: pull_request
+
```

Comment:
I'd limit the scope of this workflow to either a directory or by file extensions:
```
on: 
  pull_request:
    paths:
      - apps/utils/**
```

Reply:
Great suggestion! Done.